src/bin/ion/commands/beta/symtab/filter.rs

```diff
@@ -21,7 +21,6 @@ impl IonCliCommand for SymtabFilterCommand {
     fn configure_args(&self, command: Command) -> Command {
         command.with_input()
             .with_output()
-            .with_compression_control()
```

Comment:
🗺️ Prior to this PR, commands needed to explicitly opt into supporting the `--no-auto-decompress` flag. Forgetting to do so would cause the program to ugly-crash when the flag was specified. Now the flag is supported universally by default.